system/tests/utils.py

```diff
@@ -0,0 +1,5 @@
+class PID(object):
```

Comment:
What does PID stand for? Project ID? Reading the class name for the first time process ID came up. Can you rename it to ProjectID instead?